src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Config/ConditionalBlockCommentConfig.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.TemplateEngine.Core.Contracts;
+using Microsoft.TemplateEngine.Core.Operations;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects.Config
+{
+    public static class ConditionalBlockCommentConfig
+    {
+        public static List<IOperationProvider> ConfigureFromJObject(JObject rawConfiguration)
+        {
+            string startComment = rawConfiguration.ToString("startComment");
```

Comment:
I think "comment" is wrong for these in general, as we support reading effectively "running" preprocessors - maybe this should be "startSyntax"/"endSyntax" or something along those lines (similar for single line setups)